docs/03-privacy/privacy-incident-response.md

```diff
@@ -0,0 +1,929 @@
+* Contractors must comply with Department policies and follow all aspects of the FAR clauses
+and/or privacy provisions that are included in the contract, such as allowing Department
+inspection provisions and post-incident activities.
+
+## Notification for a Privacy Incident
+
+In the case of a major privacy incident, once the Component Information Security Officer has
+conducted the risk assessment and has identified potential mitigations, recommendations should be
+provided to the Bixal Director of Technology in the case of a major privacy incident and the privacy
+incident report should be updated.
+In the context of a major privacy incident, and as part of the effort to ensure substantive and
+effective notification, the Director of Technology will solicit input from the BISO, and the Component Information Security Officer in assessing the following issues:
+
+* Timeliness of the Notification
+* Source of the Notification
+* Contents of the Notification
+* Means of Providing the Notification
+* Who Receives Notification: Public Outreach in Response to a Privacy Incident
+
+After receiving input from the Component Information Security Officer, the Bixal Director of
+Technology provides recommendations to the BISO and Component Information Security Officer regarding
+issuance of notification to affected individuals, including timeliness, contents, means, sources,
+and general appropriateness of notification; and elevates matters to the BISO
+if the component Information Security Officer, component stakeholders, or Federal Government requires
+additional guidance or to resolve conflicts.
+In the case of a minor privacy incident, once the Component Information Security Officer has conducted
+the risk assessment and identified potential mitigations, the Privacy Incident Report should be
+updated. The assessment regarding notification to the affected individual(s) will take into account
+many, if not all, of the same issues considered in notification for major privacy incidents.
+
+The Privacy Incident Report should also detail a recommendation regarding notification to
+affected individuals.
+
+1. Timeliness of Notification for a Privacy Incident
+
+   Before notification may be issued, Bixal must first determine the scope of the privacy incident,
+   and if applicable, restore the reasonable integrity of the system or information compromised.
+   Affected individuals shall be notified without unreasonable delay following the discovery of a
+   major privacy incident, consistent with the needs of law enforcement and national security,
+   and any measures necessary for Bixal to assess the scope of the privacy incident and
+   implement containment measures.
+
+   In some circumstances, law enforcement or national security considerations may require a
+   delay in notification if it impedes the investigation of the privacy incident, as permitted by
+   OMB Memorandum M-17-12.28 Decisions to delay notification should be made by the Bixal
```

Comment:
Link to the memorandum: https://www.osec.doc.gov/opog/privacy/memorandums/omb_m-17-12.pdf ?